### ISSUE TITLE
Edge rendering broken

### ISSUE THREAD

**Issue by mgth** [realthunder/FreeCAD#898]
Using Tip 2023.10.18 in windows 
FreeCAD-Link-Tip-Win-x86_64-py3.11-20231018.7z 

Edge rendering is ok when mouse is in viewport, but completely broken when the pointer leave the zone :

<img width="766" alt="Capture d'écran 2023-10-20 114702" src="https://github.com/realthunder/FreeCAD/assets/2568124/0b3dde94-873b-4fea-ba39-dc1d913b8e14">

<img width="738" alt="Capture d'écran 2023-10-20 114802" src="https://github.com/realthunder/FreeCAD/assets/2568124/6a150b1f-122e-4eb1-ae0e-528fe1cfbe6e">

**Comment by mgth** [comment]
After some testing only auto/distributed render cache is affected.
Centralized and experimental are ok.

**Comment by realthunder** [comment]
Could you please post the file, or maybe reproduce the problem with a simplified one?

**Comment by mgth** [comment]
yes, on simple cube the vertex joins wrong corners :
[cube.zip](https://github.com/realthunder/FreeCAD/files/13068395/cube.zip)
![image](https://github.com/realthunder/FreeCAD/assets/2568124/f127ef9a-58ed-4f6b-967b-11608fd8562c)

**Comment by realthunder** [comment]
I can't reproduce the problem. Could you please send me your user config, `Tools -> Edit parameters -> Export`.

**Comment by mgth** [comment]
[mgth.zip](https://github.com/realthunder/FreeCAD/files/13162436/mgth.zip)

**Comment by alexawka** [comment]
I have encountered same problem.
When rotating object, edges get jumbled up. When hovering or clicking on object - render fixes itself until next rotation.
Changing Render Cache to Experimental fixed the issue.
FreeCad official does not seem to have same issue.

PC is Windows 11 and AMD GPU with latest drivers.
Prior on two Windows 10 and NVidia GPU it was not encountered.

Maybe windows version of gpu drivers are at fault?

**Comment by realthunder** [comment]
Yes, I can reproduce this on an AMD Windows machine, which leads me to think it is an OpenGL driver compatibility problem. I have yet to find any fix for this. Meanwhile, please try to change `Preference -> Display -> Render caching` to `Experiemental`. It works fine on my machine.
